/*
 * Copyright 2022 Google LLC.
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     https://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

syntax = "proto2";

package yggdrasil_decision_forests.dataset.proto;

// See dataset/formats.h for the definition of the formats.

// Supported dataset formats.
enum DatasetFormat {
  INVALID = 0;
  FORMAT_CSV = 1;
  reserved 2, 3, 4, 6;
  FORMAT_TFE_TFRECORD = 5;
  FORMAT_TFE_TFRECORDV2 = 8;
  FORMAT_TFE_TFRECORD_COMPRESSED_V2 = 9;
  FORMAT_PARTIAL_DATASET_CACHE = 7;
}
